breadcord/bot.py

```diff
@@ -260,5 +267,5 @@ def save_settings(self, file_path: str | PathLike[str] | None = None) -> None:
         _logger.info(f'Saving settings to {path.as_posix()}')
         path.parent.mkdir(parents=True, exist_ok=True)
         output = self.settings.as_toml().as_string().rstrip() + '\n'
-        with path.open('w+', encoding='utf-8') as file:
+        with path.open('w', encoding='utf-8') as file:
```

Comment:
What's with this change here?

Reply:
Not really relevant to the PR, but I noticed that the file opening had excessive permissions, so I changed it. 
No writing is happening so `w` is just fine

Reply:
Right, that seems fair. I would recommend testing the code out before merging though since we've accumulated quite a few changes, and I don't want to shoot myself in the foot a la Chesterton's Fence